tests/03_db/test_unit_db_models.py

```diff
@@ -930,3 +932,26 @@ async def test_project_relationships(db):
     await db.refresh(ds3)
     assert wf3.project.name == proj.name
     assert ds3.project.name == proj.name
+
+
+async def test_timestamp(db):
+    p = Project(name="project")
+    assert p.timestamp_created is not None
+    assert p.timestamp_created.tzinfo == datetime.timezone.utc
+    assert p.timestamp_created.tzname() == "UTC"
+
+    db.add(p)
+    await db.commit()
+    db.expunge_all()
+
+    query = await db.execute(select(Project))
+    project = query.scalars().one()
+
+    DB_ENGINE = Inject(get_settings).DB_ENGINE
+
+    if DB_ENGINE == "sqlite":
```

Comment:
add comment like:

Sqlite encodes `datetime` objects as strings; therefore when extracting a timestamp from the db, it is not timezone-aware by default.